.gitignore

```diff
@@ -5,6 +5,7 @@ __pycache__
 .vscode
 *.pyc
 *.snap
+*.swp
```

Comment:
Should probably be *.sw? to cover all possible extensions, for example if you edit the same file twice it will be .swp and .swo